### ISSUE TITLE
Explore Metrics: On smaller laptops, the sticky metric graph is too big

### ISSUE THREAD

**Issue by matryer** [grafana/grafana#96908]
### What happened?

On my 13" laptop, there isn't really enough room to work.

![Image](https://github.com/user-attachments/assets/dd0af526-9f5b-4752-b156-3aa6431fb5f0)

Note: after raising this, I learned that there is a setting to adjust the stickiness of that big graph. As a thought experiment, I wonder what it would look like if we removed that setting and decided ourselves, what would we do?

### What did you expect to happen?

I'd expect to be able to scroll past it. Maybe the big graph shrinks and sticks per [Torkel's idea](https://raintank-corp.slack.com/archives/C0630AYC4SY/p1710349139425779?thread_ts=1710345499.399909&cid=C0630AYC4SY).

### Did this work before?

No.

### How do we reproduce it?

1. Buy a small laptop or resize your browser
2. Go to Explore Metrics and begin a new investigation
3. Select a metric by clicking the Select button
4. Notice the big graph is biiig

### Is the bug inside a dashboard panel?

No.

### Environment (with versions)?

ops

### Grafana platform?

I use Grafana Cloud

**Comment by catherineymgui** [comment]
I don't think we should just remove the setting without an alternative option.

We discussed alternatives before! Another option is to add a resize splitter with scrolling of the tabs (like in panel edit mode). See this thread where we discussed that option: https://raintank-corp.slack.com/archives/C0630AYC4SY/p1710345499399909

**Comment by bohandley** [comment]
Closing this as the metric scene panel is no longer sticky and the setting is removed
https://github.com/grafana/metrics-drilldown/pull/624
